### PR TITLE
Fix notification flood on android

### PR DESCRIPTION
**Pull request by dab246**
## Issues

- [x] Sounds and vibrates multiple times when there are multiple notifications at the same time
- [x] Notification unstable when terminated
- [x] ADR for noti `silent`
- [ ] Unit test notification??

## Solution

- Set 
  - `silent=true` for all notifications with id `EmailId` 
  - `silent=false` for notifications with id `GroupdId`.

## Demo

1. Notifications when  `disconnected` & `reconnected`:

https://github.com/linagora/tmail-flutter/assets/80730648/c44195cf-753d-496b-94b8-fd9ab0c45953

2. Notification unstable when terminated:

- Before: 

https://github.com/linagora/tmail-flutter/assets/80730648/d759fd90-2013-4632-b51d-aa149a6d30b2

- After: 

https://github.com/linagora/tmail-flutter/assets/80730648/a35fad3b-ee5f-43c3-b8e0-d3bd51d1c74a

### CODE REVIEW

**Comment by tddang-linagora** [comment]
Is it possible to demo the fix?

**Comment by dab246** [comment]
### Note

- [ ] Delete `group_id` notification after logout

File: lib/features/push_notification/presentation/controller/fcm_message_controller.dart [hunk 1]
```diff
@@ -227,76 +213,98 @@ class FcmMessageController extends FcmBaseController {
...
   void handleFailureViewState(Failure failure) {
     log('FcmMessageController::_handleFailureViewState(): $failure');
-    _clearPayloadData();
   }
 
   @override
   void handleSuccessViewState(Success success) {
     log('FcmMessageController::_handleSuccessViewState(): $success');
-    if (success is GetAuthenticatedAccountSuccess) {
-      _handleGetAuthenticatedAccountSuccess(success);
```
Comment (hoangdat) [inline lib/features/push_notification/presentation/controller/fcm_message_controller.dart:298]: why don't need to `getAuthenticatedAccount`?

Comment (dab246) [reply lib/features/push_notification/presentation/controller/fcm_message_controller.dart:298]: We just need to get its value to process on the interactor side so we don't need to check it on the controller side anymore.

**Review by hoangdat — APPROVED** [review]
- work well
- no side effect in web

**Review by tddang-linagora — APPROVED** [review]

**Pull request merged**